src/utils.js

```diff
@@ -4,6 +4,44 @@ async function asyncForEach(array, callback) {
   }
 }
 
+function getRequiredBlockConfirmations(contract) {
+  return contract.methods.requiredBlockConfirmations().call()
+}
+
+function waitForBlockConfirmations(web3, event, requiredBlockConfirmations, blockNumberProvider) {
```

Comment:
the reason why I always use  an object instead of params, so nobody  has to follow the order of params.
Please make it as an object.
```js
{web3, event, requiredBlockConfirmations, blockNumberProvider}
```